src/Jackett.Common/Definitions/animeworld.yml

```
@@ -1,49 +1,32 @@
 ---
 id: animeworld
-name: AnimeWorld
-description: "AnimeWorld (AST4u) is a GERMAN Private site for TV / MOVIES / ANIME / HENTAI"
+name: Animeworld
+description: "AnimeWorld (AW) is a GERMAN Private site for ANIME / MANGA / HENTAI"
 language: de-de
 type: private
-encoding: iso-8859-1
+encoding: UTF-8
 links:
   - https://animeworld.cx/
-legacylinks:
-  - https://www.ast4u.me/
 
 caps:
   categorymappings:
-    - {id: 35, cat: TV/Anime, desc: "Anime Movie"}
-    - {id: 36, cat: TV/Anime, desc: "Anime Serie"}
-    - {id: 37, cat: Audio/Foreign, desc: "Anime Musik"}
-    - {id: 41, cat: Books, desc: "Anime Pic & Manga"}
-    - {id: 42, cat: XXX, desc: "Hentai Movie & OVA"}
-    - {id: 43, cat: XXX, desc: "Hentai Serie"}
-    - {id: 44, cat: PC, desc: "Hentai Game"}
-    - {id: 45, cat: Movies, desc: "Cartoon Movie"}
-    - {id: 46, cat: TV, desc: "Cartoon Serie"}
-    - {id: 47, cat: TV, desc: "TV-Serie"}
-    - {id: 49, cat: TV/Documentary, desc: "Sonstiges Doku"}
-    - {id: 50, cat: Audio, desc: "Sonstiges Soundtrack"}
-    - {id: 52, cat: Movies/HD, desc: "Movie HD"}
-    - {id: 53, cat: Other, desc: "Sonstiges"}
-    - {id: 55, cat: Movies/Foreign, desc: "Movie Asia & MartialArts"}
-    - {id: 56, cat: TV/Anime, desc: "Anime OVA"}
-    - {id: 58, cat: Movies, desc: "Movie"}
-    - {id: 59, cat: Audio/Audiobook, desc: "Sonstiges Hoerspiele"}
-    - {id: 62, cat: Console, desc: "Sonstiges Game"}
-    - {id: 65, cat: Movies/BluRay, desc: "Movie BluRay"}
-    - {id: 66, cat: TV, desc: "TV-Serie BluRay"}
-    - {id: 67, cat: TV/HD, desc: "TV-Serie HD"}
-    - {id: 68, cat: TV/Anime, desc: "Anime Movie HD & BD"}
-    - {id: 69, cat: TV/Anime, desc: "Anime OVA HD & BD"}
-    - {id: 70, cat: TV/Anime, desc: "Anime Serie HD & BD"}
+    - {id: 1, cat: TV/Anime, desc: "Anime Movie"}
+    - {id: 2, cat: TV/Anime, desc: "Anime Serie"}
+    - {id: 3, cat: Audio, desc: "Anime Musik/OST"}
+    - {id: 4, cat: PC/Games, desc: "Anime Spiele"}
+    - {id: 5, cat: XXX, desc: "Hentai"}
+    - {id: 6, cat: PC, desc: "Spiele Linux"}
+    - {id: 7, cat: Other, desc: "Sonstiges"}
+    - {id: 8, cat: Movies, desc: "Filme"}
+    - {id: 9, cat: TV, desc: "Serien"}
+    - {id: 10, cat: PC/Games, desc: "Spiele"}
+    - {id: 11, cat: Audio, desc: "Musik"}
+    - {id: 12, cat: Books, desc: "Mangas"}
 
   modes:
     search: [q]
-    tv-search: [q, season, ep]
-    movie-search: [q]
-    music-search: [q]
-    book-search: [q]
+    tv-search: [q, season, ep, imdbid, tvdbid]
+    movie-search: [q, imdbid, tmdbid]
 
```

given  that they have cats 11 and 12 I suggest retaining the music and audio search modes
```suggestion
    music-search: [q]
    book-search: [q]
    
```